### PR TITLE
Implement exists & existsSync.

### PR DESCRIPTION
**Pull request by hyunjunekim**
For #28.
Rebased it.
Closed #317 pull request. Because long time ago, removed my repository.
Thank you.

### CODE REVIEW

**Comment by hyunjunekim** [comment]
https://github.com/Samsung/iotjs/pull/317

**Comment by LaszloLango** [comment]
The commit message misses the IoT.js-DCO-1.0-Signed-off-by line. Please add that to your commit.

**Comment by hyunjunekim** [comment]
Added the IoT.js-DCO-1.0-Signed-off-by line.

**Comment by LaszloLango** [comment]
@hyunjunekim, thanks. LGTM
@akiss77, @zherczeg please review.

File: test/run_pass/test_fs_existssync.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+{
+  var filePath = "";
+
+  var result = fs.existsSync(filePath);
+  assert.equal(result, false);
+}
+
+{
+  var filePath = " ";
```
Comment (zherczeg) [inline test/run_pass/test_fs_existssync.js]: Just a note, ECMAScript has only closure scope, so all of these filePath variables are the same variable.

**Comment by zherczeg** [comment]
LGTM

**Pull request merged**